src/status_im/ui/screens/keycard/components/keycard_animation.cljs

```diff
@@ -0,0 +1,338 @@
+                                            :duration 1000
+                                            :easing   card-easing})
+        card-loop        (animation/anim-loop
+                          (animation/anim-sequence
+                           [(animation/timing card
+                                              {:toValue  #js {:x -30
+                                                              :y 30}
+                                               :duration 1000
+                                               :easing   card-easing})
+                            (animation/timing card
+                                              {:toValue  {:x 45
+                                                          :y 65}
+                                               :duration 1000
+                                               :delay    2000
+                                               :easing   card-easing})
+                            (animation/timing card
+                                              {:toValue  #js {:x -30
+                                                              :y 105}
+                                               :duration 1000
+                                               :delay    2000
+                                               :easing   card-easing})
+                            (animation/anim-delay 2000)]))
+        phone-entrace    (animation/anim-sequence
+                          [(animation/anim-delay phone-enter-at)
+                           (animation/parallel
+                            [(animation/timing card-scale
+                                               {:toValue  0.528
+                                                :duration 1000
+                                                :easing   card-easing})
+                             (animation/timing phone
+                                               {:toValue  1
+                                                :duration 1000
+                                                :easing   card-easing})
+                             card-loop])])
+        circles          (animation/anim-loop
+                          (animation/parallel
+                           [(animation/anim-sequence
+                             [(circle-animation small 1 0)
+                              (circle-animation small 0 0)])
+                            (animation/anim-sequence
+                             [(circle-animation medium 1 200)
+                              (circle-animation medium 0 0)])
+                            (animation/anim-sequence
+                             [(circle-animation big 1 400)
+                              (circle-animation big 0 0)])]))
+        animation        (animation/parallel
+                          [resets
+                           phone-entrace
+                           circles])]
+    (reset! state :init)
+    ;; TODO: Think how to improve that, this creates desync with animation
```

Comment:
please add you name to TODOs, it simplifies finding TODO's author after code "migrations"

```suggestion
    ;; TODO(Ferossgp): Think how to improve that, this creates desync with animation
```